docs(schema): improve Cube View documentation (#5463)
cube-js/cube.js: docs/content/Schema/Reference/view.mdx

subCategory: Reference
---

Views sit on top of the data graph of cubes and create a facade of your whole
data model with which data consumers can interact. They provide single place to
define metrics, manage governance/data access, and control ambiguous join paths.

> [![](https://mermaid.ink/img/pako:eNp1kt9LwzAQx_-VIzL2skKH-LAIgnWKbyKbvrR7yJqrDWubkh9uY-x_N0m13XQ2FO4-9819014OJJccCSWj0QFEIwyFA4yLSm7zkikzDmlu1Se6cLxmWugxHP0ajbImaz4Ua0tYzl1sXi2qvQ_8q-26q70L3GpPwD0-eaxbs08DXp3y-3RRirZFDi-KozovJumz-HDNWGURHqw2sv6RYMOzxut-GT_YNfbGPumMA16d8vv0l6GHSfqm_3EI3wlRFN11577AEs9CsWPB5gJLur5hywVlcqbs3POKaT3HApbCVAiFqCp6NbueTYt8Fk9gLXeRLhmXWwo37Q6mcbubQC4rqSh86-L4tu_U_aau2QkN4xloOGXv_MQ2-Nc9du7aKLnBaCu4KSl4m37jMIJh_2m5vxpDmUyIm3LNBHcX9ODFGTEl1pgR6kLO1CYjWXN0OmaNXOybnFCjLE6IbTkzOBfM3YWa0IJVGo9fIZX7Vg)](https://mermaid.live/edit#pako:eNp1kt9LwzAQx_-VIzL2skKH-LAIgnWKbyKbvrR7yJqrDWubkh9uY-x_N0m13XQ2FO4-9819014OJJccCSWj0QFEIwyFA4yLSm7zkikzDmlu1Se6cLxmWugxHP0ajbImaz4Ua0tYzl1sXi2qvQ_8q-26q70L3GpPwD0-eaxbs08DXp3y-3RRirZFDi-KozovJumz-HDNWGURHqw2sv6RYMOzxut-GT_YNfbGPumMA16d8vv0l6GHSfqm_3EI3wlRFN11577AEs9CsWPB5gJLur5hywVlcqbs3POKaT3HApbCVAiFqCp6NbueTYt8Fk9gLXeRLhmXWwo37Q6mcbubQC4rqSh86-L4tu_U_aau2QkN4xloOGXv_MQ2-Nc9du7aKLnBaCu4KSl4m37jMIJh_2m5vxpDmUyIm3LNBHcX9ODFGTEl1pgR6kLO1CYjWXN0OmaNXOybnFCjLE6IbTkzOBfM3YWa0IJVGo9fIZX7Vg)

Author: Will be replacing this image with a nicer one

subCategory: Reference
---

Views sit on top of the data graph of cubes and create a facade of your whole

> data model with which data consumers can interact. They provide single place to

Reviewer: ```suggestion
data model that data consumers can interact with. They provide a single place to
```

GROUP BY 2
```

Views also do not define any pre-aggregations, instead they re-use
pre-aggregations defined in the underlying cubes used by the view.

## Parameters

### <--{"id" : "Parameters"}--> description

> A description of the view to allow your team to better understand what its

Reviewer: ```suggestion
A description of the view allows your team to better understand what its
```

purpose is. It is a very simple and yet useful tool that gives a hint to

> everyone and ensures data is interpreted correctly by users.

Reviewer: ```suggestion
everyone and ensures that data is interpreted correctly by users.
```
